File: I-DataStructures/1-Array/array.go
package main

import "fmt"

func main() {
	fmt.Println("Arrays in Go")
	var fruits [5]string
	fruits[0] = "Apples"
	fruits[1] = "Bananas"
	fruits[2] = "Oranges"
	fruits[3] = "Peaches"
	fruits[4] = "Berries"
	fmt.Println("Before: " + fruits[0])
	fruits[0] = "Pineapples"
	fmt.Println("After: " + fruits[0])
}

File: I-DataStructures/1-Array/array.java
public class array {
    public static void main(String[] args)
    {
        System.out.println("Arrays in Java");
        String[] fruits = {"Apples", "Bananas", "Oranges", "Peaches", "Berries"};
        System.out.printf("Before: %s\n", fruits[0]);
        fruits[0] = "Pineapples";
        System.out.printf("After: %s\n", fruits[0]);
    }
}